[Alerts][Docs] Added API documentation for alerts plugin (#91067)
elastic/kibana: docs/api/alerts/mute.asciidoc

[[alerts-api-mute]]
=== Mute alert instance API
++++
<titleabbrev>Mute alert instance</titleabbrev>
++++

Mute {kib} alert instance.

[[alerts-api-mute-request]]
==== Request

`POST <kibana host>:<port>/api/alerts/alert/<id>/alert_instance/<alert_instance_id>/_mute`

[[alerts-api-mute-path-params]]
==== Path parameters

`id`::

>   (Required, string) The alert ID that you want to mute a single instance.

Reviewer: ```suggestion
  (Required, string) The Alert ID of the Alert whose instance we wish to mute.
```
